factory/__main__.py

```diff
@@ -15,74 +14,84 @@ def main():
     signal.signal(signal.SIGINT, signal_handler)
     console = Console()
 
+    gruvbox_factory = GoNord()
+    gruvbox_factory.reset_palette()
+    add_gruvbox_palette(gruvbox_factory)
+
     # Checks if there's an argument
     if len(sys.argv) > 1:
-        image_file = fromCommandArgument(console)
+        image_paths = fromCommandArgument(console)
     else:
-        image_file = fromTui(console)
-    process_image(image_file, console)
+        image_paths = fromTui(console)
 
+    for image_path in image_paths:
+        if os.path.isfile(image_path):
+            process_image(image_path, console, gruvbox_factory)
+        else:
+            console.print(
+                f"❌ [red]We had a problem in the pipeline! \nThe image at '{image_path}' could not be found! \nSkipping... [/]"
+            )
+            continue
 
 
 # Gets the file path from the Argument
 def fromCommandArgument(console):
 
     command_parser = argparse.ArgumentParser(
-        description='A simple cli to manufacture a gruvbox themed wallpaper.'
+        description="A simple cli to manufacture gruvbox themed wallpapers."
+    )
+    command_parser.add_argument(
+        "Path", metavar="path", nargs="+", type=str, help="The path(s) to the image(s)."
     )
-    command_parser.add_argument('Path',
-                       metavar='path',
-                       type=str,
-                       help='The path to the image.')
     args = command_parser.parse_args()
 
-    if args.Path is None:
-        console.print('❌ [red]We had a problem in the pipeline! Make sure you\'re in the same path of the image you want to convert! [/]')
-        sys.exit(0)
-    else:
-        return args.Path
+    return args.Path
 
 
 # Gets the file path from user input
 def fromTui(console):
 
-    console.print(Panel('🏭 [bold green] Gruvbox Factory [/] 🏭', expand=False, border_style='yellow'))
-    console.print('⚠️ WARNING ⚠️\n[italic]make sure you\'re in the same directory of the image you want to convert [/]\n')
-    return console.input('🖼️ [bold yellow]Which image do you want to manufacture?[/] ')
+    console.print(
+        Panel(
+            "🏭 [bold green] Gruvbox Factory [/] 🏭", expand=False, border_style="yellow"
+        )
+    )
 
-def process_image(image_file, console):
-    gruvbox_factory = GoNord()
-    gruvbox_factory.reset_palette()
-    add_gruvbox_palette(gruvbox_factory)
-    try:
-        image = gruvbox_factory.open_image(image_file)
-    except:
-        console.print('❌ [red]We had a problem in the pipeline! Make sure you\'re in the same path of the image you want to convert or the Path is correct! [/]')
-        sys.exit(0)
-    console.print('🔨 [yellow]manufacturing your gruvbox wallpaper...[/]')
-    
+    return [
+        os.path.expanduser(path)
+        for path in console.input(
+            "🖼️ [bold yellow]Which image(s) do you want to manufacture?[/] "
```

Comment:
Could you add something like "Which image(s) do you want to manufacture? (images paths separated by spaces)"?

Reply:
Sure! The prompt is changed now.